elizhahn/static-comp
# Refactor/clean up

## Is this a fix or a feature?
- fix

## What is the change?


## What does it fix?

- cleans up formatting
- deleted unnecessary rules
- changed some rules to use same properties for consistency
- Added an activated rule that would be used for when a user is in a certain navigation view
- fixed typo in title of page

## Where should the reviewer start?
- index.html
- styles.css
## How should this be tested?
- clone down the repo
- open index.html
- view the page in Chrome, Safari, Firefox 
- change the screen size to see responsive design

- The page should match the comp on these three browsers and be responsive

## Screenshots (if appropriate):
